### DISCUSSION TITLE
CentOS 8 compiling error - cannot install

### DISCUSSION THREAD

**Discussion by alansenairj** [osm-search/Nominatim#2546] [Q&A]
**Describe the bug**

error compiling nomitatim in CentOS 8. 

First error: ( I found  something about Oracle's Cloud Error on CentOS 8 related bug)
[opc@instance-20211208-1949 ~]$ sudo dnf -qy module disable postgresql
Failed to set locale, defaulting to C.UTF-8
Failed loading plugin "osmsplugin": No module named 'librepo'

[100%] Running external makefile /usr/pgsql-12/lib/pgxs/src/makefiles/pgxs.mk
/root/Nominatim/module/Makefile:11: warning: overriding recipe for target 'install'
/usr/pgsql-12/lib/pgxs/src/makefiles/pgxs.mk:241: warning: ignoring old recipe for target 'install'
gcc: error: /usr/lib/rpm/redhat/redhat-hardened-cc1: No such file or directory
make[3]: *** [<builtin>: nominatim.o] Error 1
make[2]: *** [module/CMakeFiles/nominatim_lib.dir/build.make:73: ../module/dummy] Error 2
make[1]: *** [CMakeFiles/Makefile2:1023: module/CMakeFiles/nominatim_lib.dir/all] Error 2
make: *** [Makefile:146: all] Error 2

**To Reproduce**

**Software Environment (please complete the following information):**
- Nominatim version: latest
- Postgresql version: 12
- Postgis version:
- OS: CentOS 8

**Hardware Configuration (please complete the following information):**
- RAM: 16
- number of CPUs: 4
- type and size of disks:
- bare metal/AWS/other cloud service: Oracle Cloud

**Postgresql Configuration:**

**Additional context**

**Comment by lonvia** [comment]
Your gcc is not installed correctly. This is not an issue with nominatim but with your system installation.

Please do not use the issue tracker to ask installation questions. Moving to discussion section.
